BUILDING.md

```diff
@@ -544,12 +544,13 @@ Optional requirements to build the MSI installer package:
 
 * The [WiX Toolset v3.11](https://wixtoolset.org/releases/) and the
   [Wix Toolset Visual Studio 2019 Extension](https://marketplace.visualstudio.com/items?itemName=WixToolset.WixToolsetVisualStudio2019Extension).
+* The [WiX Toolset v3.14](https://wixtoolset.org/releases/) if
+  building for Windows 10 on ARM (ARM64).
 
 Optional requirements for compiling for Windows 10 on ARM (ARM64):
 
-* ARM64 Windows build machine
-  * Due to a GYP limitation, this is required to run compiled code
-    generation tools (like V8's builtins and mksnapshot tools)
+* ARM64 or x64 Windows build machine
+  * ARM64 builds can be created on a x64 host by running `.\vcbuild arm64`
```

Comment:
```suggestion
```

Just a small detail: AFAICT, native compilation does not work (would be happy to be proved wrong though). I think these lines about the build machine can simply be removed, if you agree.

Reply:
Just tested and you're right. While the build process starts on my Surface Pro X (ARM64), it errors in several places and eventually stops. 😞